entries/w/wetransfer.com.json

```
@@ -0,0 +1,15 @@
+{
+    "WeTransfer": {
+        "domain": "wetransfer.com",
+        "tfa": [
+            "totp"
+        ],
+        "documentation": "https://help.wetransfer.com/hc/en-us/articles/360056666872",
+        "recovery": "https://help.wetransfer.com/hc/en-us/articles/360056666872#h_01EXEFX7G5DKD2GHYDGWH1S0C7",
+        "categories": [
+            "creativity",
+            "social",
+            "other"
```

So it's a file uploading site?
The __Sync__ in Backup & Sync makes that category a better fit for these types of sites.
```suggestion
            "backup"
```

---

They have [other products](https://wetransfer.com/explore/products) too, specifically [Paste](https://wetransfer.com/paste), which shares a common authentication backend with WeTransfer.

---

Okay so backup and creativity?

---

Yup. I'll update the PR.